src/main/resources/db/migration/V86__create_opiskeluoikeushistoria_virheet_table.sql

```diff
@@ -0,0 +1,9 @@
+-- Table Definition ----------------------------------------------
+
+CREATE TABLE opiskeluoikeushistoria_virheet (
+    id integer GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    aikaleima time with time zone NOT NULL DEFAULT now(),
+    opiskeluoikeus jsonb NOT NULL,
```

Comment:
Pitäisiköhän näissä sallia NULLit? Lähinnä pohdin skenaariota, että jos datan luonnissa menee jotain pieleen, voiko tänne kuitenkin päätyä NULL-datalla, mikä sitten failaa koko opiskeluoikeuden siirron tietokantavirheeseen?

Reply:
Sellainen tilanne ei kai pitäisi olla mahdollinen? Ymmärtääkseni silloin kyllä siirto epäonnistuu jo rakenteen validaatioon. Ainoa tilanne missä näitä virheitä voi generoitua on PostgresOpiskeluoikeusRepository::updateAction -kutsun myötä.